src/routes/userRoute.ts

```diff
@@ -0,0 +1,160 @@
+import { Hono } from 'hono';
+import { PrismaClient } from '@prisma/client';
+import { validator } from 'hono/validator';
+import {
+  schemaEmail,
+  schemaId,
+  schemaLogin,
+  schemaPassword,
+  schemaUser,
+} from '../models/userSchema';
+import bcrypt from 'bcrypt';
+
+const prisma = new PrismaClient();
+
+export const userRoute = new Hono();
+
+userRoute.get('/', async (c) => {
+  return c.text('Hello!');
+});
+
+userRoute.get('/users', async (c) => {
+  const allUsers = await prisma.user.findMany();
+  return c.json({ data: allUsers, ok: true });
+});
+
+userRoute.get(
+  '/users/:id',
+  validator('param', (value, c) => {
+    const parsedId = schemaId.safeParse(value.id);
+
+    if (!parsedId.success) {
+      return c.json({ error: 'Invalid ID. Not Found.', ok: false }, 404);
+    }
+    return { id: value.id };
```

Comment:
Please use `parsedId.data` instead of `value.id` to get type number in the controller